units/ldconfig.service

```diff
@@ -10,9 +10,8 @@ Description=Rebuild Dynamic Linker Cache
 Documentation=man:ldconfig(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
-After=systemd-remount-fs.service
+After=local-fs.target
 Before=sysinit.target shutdown.target systemd-update-done.service
-ConditionNeedsUpdate=|/etc
 ConditionFileNotEmpty=|!/etc/ld.so.cache
```

Comment:
I was wondering whether we should leave this as a trigger condition. I think yes. As long as there's only one condition, there's no difference. But having it as a trigger condition makes it easier to override or extend, for example by a drop-in which restores previous behaviour.